package.json

```diff
@@ -58,10 +58,10 @@
   ],
   "dependencies": {
     "global": "^4.3.2",
-    "video.js": "^6 || ^7"
+    "video.js": "^7 || ^8"
```

Comment:
I believe this should still be compatible with `^6` now that we fall back to `mergeOptions`, so we might as well keep it in there.

But I think #142 is probably a big enough change to warrant a major version of this plugin, so maybe the best move is just to bite the bullet and make it compatible with 8+ only and increment the major version?